### ISSUE TITLE
ListView: Test site as a code generator?

### ISSUE THREAD

**Issue by staxmanade** [winjs/winjs#1099]
# TL;DR

Would be great if  [ListView Test](http://try.buildwinjs.com/pages/listview/options/default.html) could be used as a sort-of code generator for **all** of the options specified on the left. Where the generated code can be both a sample in HTML and in plain JS.

# Full Summary

I took a look at the [ListView Test](http://try.buildwinjs.com/pages/listview/options/default.html) and didn't initially notice that the `Documentation` button on the right (when clicked) would show sample code on the most recently selected option on the left.

I think it would be great use the options on the left to configure all of the various properties for a grid (like it does now) and visually see my options (like it does now); however, once complete, instead of showing just the last selected item and some sample JS code in the documentation area, would be great if the tool could spit out a code-generated version of a ListView with **all** of the options specified.

This would allow (those like myself) who can't remember all of the magic-strings and syntax around creating a grid, and could easily design-it in the view and copy-paste the generated code. (Copy-Paste-Inheritance is great) ![copy paste everywhere](http://zillionthoughts.com/wp-content/uploads/2015/01/copypaste-copypaste-everywhere.jpg)

Thoughts?

**Comment by jayrenn** [comment]
This is a great idea! I have added this request to our internal tracking.
